src/economy.cpp

```diff
@@ -133,8 +157,9 @@ Money CalculateCompanyValue(const Company *c, bool including_loan)
 		}
 	}
 
-	/* Add real money value */
 	if (including_loan) value -= c->current_loan;
+
+	/* Add real money value */
```

Comment:
why this comment move?

Reply:
Comment applies to line below the comment. There was an unnecessary space.